src/Product.php

```diff
@@ -978,4 +979,52 @@ protected function processTexts(array $data)
 
         return $this;
     }
+
+    /**
+     * @param array $data
+     * @return $this
+     */
+    protected function processTags($data)
+    {
+        if (!is_array($data) || empty($data)) {
+            $this->handleEmptyData();
+            return $this;
+        }
+
+        $keywords = $this->getField('keywords');
+        if ($keywords != '') {
+            $keywords = $keywords . ',';
+        }
+
+        foreach ($data as $tag)
+        {
+            if ($tag['tagType'] != 'variation') {
+                continue;
+            }
+
+            $correctTagName = null;
+
+            if (!empty($tag['tag']['names'])) {
+                foreach ($tag['tag']['names'] as $tagName) {
+                    if ($tagName['tagLang'] === strtolower($this->getLanguageCode())) {
+                        $correctTagName = $tagName['tagName'];
+                    }
+                }
+            }
+
+            if (!$correctTagName) {
+                $correctTagName = $tag['tag']['tagName'];
+            }
+
+            $keywords = $keywords . $correctTagName . ',';
+        }
+
+        if (substr($keywords, -1) === ',') {
```

Comment:
I would suggest pushing the tag names into an array and than use `implode` so you don't have to take care of trailing commas.